plugins/backstage-plugin-coder/src/components/CoderWorkspacesCard/CreateWorkspaceLink.tsx

```diff
@@ -45,22 +56,51 @@ export const CreateWorkspaceLink = ({
   tooltipProps = {},
   ...delegatedProps
 }: CreateButtonLinkProps) => {
-  const styles = useStyles();
   const { workspacesConfig } = useWorkspacesCardContext();
+  const canCreateWorkspace = Boolean(workspacesConfig.creationUrl);
+  const styles = useStyles({ canCreateWorkspace });
 
   return (
-    <Tooltip ref={tooltipRef} title={tooltipText} {...tooltipProps}>
+    <Tooltip
+      ref={tooltipRef}
+      title={
+        canCreateWorkspace ? tooltipText : 'Please add a template name value'
```

Comment:
I at first did not notice the extra help below, should we maybe say "check below for more information"?  Alternatively, if we moved the accordion closer to the button that could help.  I might just be particularly blind though.

Reply:
Yeah, good call. I was a little worried about putting too much text in the tooltip, but after looking at MUI's docs again, it looks like you can use fully-customizable HTML instead of plain text

Will update things to make the relationship more obvious

Reply:
Updated the UI. Tried to add some subtle color to the make the accordion stand out more, but it either looked super ugly, or started to break from the default Backstage styling

I'm going with a longer tooltip message for now; will reconsider adding color if we get more feedback

Reply:
Sweet!